### PR TITLE
Reloved Bahmni and OpenHMIS modules

### PR DESCRIPTION
**Pull request by valvijo**

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -109,21 +79,6 @@
       "filename": "htmlwidgets-1.7.2.omod"
     },
     {
-      "name": "ID Generation",
```
Comment (ningosi) [inline modules.json:112]: I think we need this module for our identifier creation? Or how are identifier created?

File: modules.json [hunk 2]
```diff
@@ -234,24 +178,6 @@
...
-      "filename": "openhmis.commons-3.4.3.omod"
-    },
-    {
-      "name": "OpenHMIS Backbone Forms",
-      "version": "2.4.0",
-      "url": "https://modules.openmrs.org/modulus/api/releases/1315/download/openhmis.backboneforms-2.4.0.omod",
-      "filename": "openhmis.backboneforms-2.4.0.omod"
-    },
-    {
       "name": "Jasper Report",
```
Comment (ningosi) [inline modules.json:181]: This one was required for the Phamacy module as a result of openHMIS modules. Apart from those, we might NOT need it for now

**Comment by ningosi** [comment]
The pharmacy modele that was included here had dependencies on OpenHMIS modules, and since they posted some issues in starting them, @steliomo modified that module and removed all those, we therefor e need to rebuilt that latest version and replace it with the current one. @valvijo

File: modules.json (hunk 1 above)
Comment (valvijo) [reply modules.json:112]: This is an id autogen, our identifiers are set manually, no dependency at all with this module, not now.

File: modules.json (hunk 2 above)
Comment (valvijo) [reply modules.json:181]: Ok, will fix this

Comment (valvijo) [reply modules.json:181]: Actually we don't need it because our pharmacy should work without OpenHMIS dependencies

**Pull request merged**